### ISSUE TITLE
abeni AI

### ISSUE THREAD

**Issue by aabeni771-ai** [tensorflow/tensorflow#102354]
# Neural network 
#102313 

import
from  import layers
import as np

# 1. Data qopheessuu (fakkeenya data x fi y)
# x = input, y = output (fakkeenya  y = 2x + 1 barachuu)
x = np.array([-1.0, 0.0, 1.0, 2.0, 3.0, 4.0], dtype=float)
y = np.array([-1.0, 1.0, 3.0, 5.0, 7.0, 9.0], dtype=float)

# 2. Model ijaarruu
model = keras.Sequential([
    layers.Dense(units=1, input_shape=[1])
])

# 3. Model leenjisuu (training)
model.compile(optimizer='sgd', loss='mean_squared_error')
model.fit(x, y, epochs=500, verbose=False)

# 4. Test gochuu
print("Tilmaama:", model.predict([10.0]))

**Comment by mihaimaruseac** [comment]
Please don't spam
